scss/_layouts_application-panels.scss

```diff
@@ -1,16 +1,28 @@
 @mixin vf-l-application-panels {
   $header-padding-bottom: $sp-unit * 6;
 
-  .p-panel,
-  .p-panel--aside {
+  .p-panel {
     background: $colors--light-theme--background-default;
     color: $colors--light-theme--text-default;
+    max-width: 100%;
     min-height: 100%; // make sure it fills whole container even when has little content
 
     &.is-dark {
       background: $colors--dark-theme--background-default;
       color: $colors--dark-theme--text-default;
     }
+
+    &.is-wide {
+      width: 720px; // ~ col-8
+    }
+
+    &.is-medium {
```

Comment:
I think we want this as the default. There is no case where we just want the content to dictate the width.